README.md

```diff
@@ -1,8 +1,7 @@
 
 # Elyra AIDevSecOps Tutorial
 
-This tutorial is used to discuss the interface between Data Science and Dev/DevOps using project templates, pipelines and bots.
-Moreover, it wants to highlight that Data Scientists are not so different from developers and DevSecOps practices and tools can be applied to MLOps ones.
+This tutorial is used to discuss the interface between data science and DevOps using project templates, pipelines and bots. It looks to highlight that data scientists are not so different from developers, and many MLOps practices and tools can be enhanced by DevSecOps techniques.
 
 The demo application used is the "hello world" for AI: MNIST Classification.
 
```

Comment:
As a first time user, I could be intimidated by the need to setup all of the following required environments. Can we mention something like "the following requirements are already setup and made available to you." ?

Reply:
when say required environment what are you referrint to @oindrillac ?

- 1. https://github.com/AICoE/elyra-aidevsecops-tutorial/blob/master/docs/source/pre-requisite.md or
- 2. https://github.com/AICoE/elyra-aidevsecops-tutorial#operate-first-open-environment

- 1. seems necessary until meteori is coming to life. Then we can add a section, if you use Meteor, skip this step, something like this. wdyt?

- 2. are targeting people with their own cluster and ODH deployed that want to run a similar tutorial

Reply:
@pacospace I'm referring to the below section which highlights the following. It might be worthwhile mentioning to what extent the following is or isnt setup for them. Looking at a list like this, as a new user, I might think I need to setup/learn Tekton or get and/or provision my own OpenShift cluster

- [Open Data Hub][3] v1.0,
- [Openshift][11] (Enterprise Kubernetes),
- Cloud Object Storage (e.g. Ceph, minio).
- [Tekton][10], used in CI/CD systems, to run pipelines created by humans or machines.
- [ArgoCD][12], used for Continuous Deployment of your applications.
- Tutorial [container image](https://quay.io/repository/thoth-station/s2i-lab-elyra?tag=ml-prague-workshop&tab=tags):

Reply:
should we make this more explicit? there is this section here: https://github.com/AICoE/elyra-aidevsecops-tutorial#operate-first-open-environment

```
Operate First Open Infrastructure environment has been selected to run this tutorial. It fullfills all the requirement stated above. If you are interested in using it, just get in touch with Operate First team, it's an open source initiative.

You can find some notes also regarding other environments in the different sections of the tutorial.
```

Reply:
Can we explicitly mention something like 

" This tutorial has the following environment requirements to be run. If you're running on Project Meteor which uses the Operate First environment, the Environment Requirements are already setup for you. Please see the Note below. "

Reply:
Perfect :) and then we will update it with the link, once available. thanks @oindrillac, @isabelizimm can you update that section please?

Reply:
:+1: